src/main/java/ch/njol/skript/expressions/ExprLevelProgress.java

```diff
@@ -101,13 +101,13 @@ public void change(final Event e, final @Nullable Object[] delta, final ChangeMo
 					return;
 			}
 			p.setLevel(Math.max(0, p.getLevel() + (int) Math.floor(c)));

```

Comment:
don't you have to use the method on the level as well

Reply:
No, because it's converted to an int, which destroys the NaN/INF value.

Reply:
Missed that bit, should be good then.